src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -468,7 +471,10 @@ TEST_F(PolicyHandlerTest, ReceiveMessageFromSDK) {
   EXPECT_CALL(app_manager_, GetNextHMICorrelationID());
   EXPECT_CALL(mock_message_helper_, CreateGetVehicleDataRequest(_, _, _));
   EXPECT_CALL(*mock_policy_manager_, PTUpdatedAt(_, _));
-  EXPECT_CALL(*mock_policy_manager_, LoadPT("", msg)).WillOnce(Return(true));
+  EXPECT_CALL(*mock_policy_manager_, LoadPT("", msg))
```

Comment:
@AKalinich-Luxoft why do we expect LoadPT with empty string?

Reply:
@LuxoftAKutsan it's a nature of this test case - we are passing empty string in `ReceiveMessageFromSDK` and expecting `LoadPT` to be called with the same argument